NeurodataWithoutBorders/matnwb
# Remove toolbox dependencies from ecephys tutorial

Fixes #445 

## Motivation

- Not useful for someone who doesn't have the statistics toolbox
- A lot more work to get working with CI.

## How to test the behavior?
Without any toolboxes installed:
```
run('tutorials/ecephys');
```
Should run without error.

## Checklist
